lib/jit_preloader.rb

```diff
@@ -25,6 +25,14 @@ def self.globally_enabled=(value)
     @enabled = value
   end
 
+  def self.query_dive_limit=(limit)
+    @dive_limit = [limit - 1, 1].max
```

Comment:
Why the -1?

Reply:
This is base on previously testing in this [post](https://github.com/clio/themis/discussions/130600). 
When dive limit is 10, MySql still not trigger the range statistics estimated when it is exactly 10.
But only when it is _less than_ 10.